security/botan/files/patch-compiler_flags.diff

```diff
@@ -1,6 +1,6 @@
---- src/build-data/cc/clang.txt.orig	2022-06-03 12:29:40.000000000 -0500
-+++ src/build-data/cc/clang.txt	2022-10-29 02:06:49.000000000 -0500
-@@ -76,9 +76,10 @@
+--- src/build-data/cc/clang.txt
++++ src/build-data/cc/clang.txt
+@@ -76,10 +76,11 @@ all!haiku,llvm -> "-pthread"
```

Comment:
I'm confused by the ending of this line.

Reply:
This diff was generated by `git diff`. Git includes additional context information at the ends of these `@@` lines. Diffs generated by `diff` don't have that.